### ISSUE TITLE
How to train the model by myself with the method

### ISSUE THREAD

**Issue by ucasiggcas** [marl/crepe#29]
Dear,
When I use 
`crepe audio_file.wav`
in piano pitches track,
the results are not good in low frequency,so I just want to train the model by myself using DCNN proposed .
Please help me,SOS
Thx

**Comment by jongwook** [comment]
Hi @ucasiggcas , I'm sorry to hear that the pitch accuracy is not satisfactory.

We didn't open-source the training scripts, as it is not in a user-friendly state to train with newer datasets.

Meanwhile, since [the model architecture is already in the code](https://github.com/marl/crepe/blob/master/crepe/core.py#L24-L81), I'd suggest following the [Keras documentation](https://keras.io/) to build your own training routine. You can then replace the model file at `{your python installation}/lib/python3.6/site-packages/crepe/model-full.h5` to use your newer model with the `crepe` command.
